ex7.1_NumPy_vector.py

```diff
@@ -0,0 +1,16 @@
+from numpy import array
+
+
+def create_vector():
+    """
+    function that create a vector with number from 0 to 20 and returns the numbers between 9 and 15 to negative once
+    :return:
+    """
+    vector = array(range(21))
```

Comment:
Always try to use numpy built in functions.
https://numpy.org/doc/stable/reference/generated/numpy.arange.html